libraries/AP_HAL_ChibiOS/hwdef/speedybeef4/hwdef.dat

```diff
@@ -145,6 +145,7 @@ define HAL_OSD_TYPE_DEFAULT 1
 ROMFS_WILDCARD libraries/AP_OSD/fonts/font*.bin
 
 # minimal drivers to reduce flash usage
-include ../include/minimal.inc
```

Comment:
Should we remove this?

Reply:
we should add minimal into minimize common in my opinion....and the get rid if it